website/search_migration/migrate.py

```diff
@@ -88,6 +93,54 @@ def migrate_nodes(index, delete, increment=10000):
             spam_flagged_removed_from_search=settings.SPAM_FLAGGED_REMOVE_FROM_SEARCH)
         logger.info('{} nodes marked deleted'.format(total_nodes))
 
+def migrate_preprints(index, delete):
```

Comment:
Search migration for prepirnts - since there are much fewer preprints, this hasn't been written in SQL.